### ISSUE TITLE
Pivot

### ISSUE THREAD

**Issue by MikeLowery1974** [Courseplay/courseplay#1163]
Question does courseplay support the irrigation pivot. I set up the course but it wouldn't activate. it maybe something i was doing wrong i don't know. thanks for any advise.

Error: Running LUA method 'mouseEvent'.
F:\FARMING//ZZZ_courseplay/start_stop.lua:76: attempt to perform arithmetic on local 'Ty' (a nil value)
Warning (LUA): Argument to method 'getTranslation' is nil
-- from =F:\FARMING//ZZZ_courseplay/start_stop.lua:75

**Comment by jhomer** [comment]
I've been messing around with those pivot irrigation for use with the soil mod.
I did remember seeing the same error but haven't seen it lately and they've been working fine with CP.
The first things I did was try to take out the collision to eliminate the "CP in traffic" -not too real but I wouldn't want to use them if they had to be moved all the time.
Also changed them to spray water.
The rigid body just can't be removed otherwise they don't even sit properly or the game don't even load.
So after some trial and error they now load perfectly and can drive through them everywhere and without the bouncing around when they first load like the original did.
I do have a few warnings for "static actor moved" and have not found a way around this yet.
And an error for monitory keyframes (or something like that) that I've been trying to fix yet-(think this may have been a problem with the originals).
Also a warning for the fuel usage cause they do not use any fuel. -minor.
I also have been working on the drive wheels and the differentials on the 200 so far and seems to be working.
Some I recorded a course like normal and drive them around in a circle and stop recording.
They will start where they should and turn on the irrigation, and they will then stop when they finish.
A couple I have only doing a 180- course recorded just the same but after the 180, hit the reverse and drive back the 180 a bit past the start point then hit forward and drive close to the start point and stop recording.
When running them, just like normal, they need to be in fertilizer/seeding mode for them to self activate.
